app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -40,6 +43,18 @@ class MainActivity : AppCompatActivity() {
     private fun setupBottomNavigationMenu(navController: NavController) {
         setupWithNavController(navigation, navController)
         setupWithNavController(navigationAuth, navController)
+
+        navigation.setOnNavigationItemReselectedListener { item: MenuItem ->
+            val hostFragment = supportFragmentManager.findFragmentById(R.id.frameContainer)
+            if (hostFragment is NavHostFragment) {
+                val currentFragment = hostFragment.childFragmentManager.fragments.first()
+                when (item.itemId) {
```

Comment:
Just check if the fragment is implementing the interface and call the function

Reply:
I have followed your guidelines and made the changes